Automattic/jetpack
# Product views: update icons to use VaultPress & Akismet icons

Fixes #29045

## Proposed changes:

Let's move from the generic cloud and bug icons to VaultPress and Akismet icons.

<img width="436" alt="image" src="https://user-images.githubusercontent.com/426388/220167782-c8b6db04-5abf-477f-b546-cff9433b80a6.png">

<img width="587" alt="image" src="https://user-images.githubusercontent.com/426388/220167821-efaa21a9-479d-4ccb-a6bf-13e28a2ce638.png">

- [ ] This may be the opportunity to change "Jetpack Backup" into "Jetpack VaultPress backup" on that screen. I think that's the name we go with these days?
- [ ] Still need to build a new "combined" icon as well:

<img width="386" alt="image" src="https://user-images.githubusercontent.com/426388/220167973-3888e243-01f3-4c90-91e8-19b0ab4ae064.png">


### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Start with a free site, connected to WordPress.com
* Go to `wp-admin/admin.php?page=jetpack#/product/akismet`
* Go to `wp-admin/admin.php?page=jetpack#/product/backup`
* Check that the icons look good.
